### PR TITLE
Changed GriffinCamp link to La Commune

### PR DESCRIPTION
**Pull request by karlsc**
Changed GriffinCamp to La Commune

### CODE REVIEW

**Comment by ziad-saab** [comment]
This is a great start. I forgot to mention that there is another "griffin camp" somewhere in the codebase. Can you find it and modify it as well please?

**Comment by karlsc** [comment]
Hi Ziad, can you tell me if this is ok now? Thanks

File: src/about/index.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h1 class="intro-section__headline">About Us</h1>
   <div class="section-content">
     <h1 class="about-page__heading">About DecodeMTL</h1>
     <p class="about-page__text">
-      We are currently running our courses out of the GriffinCamp a leading startup accelerator in downtown Montreal's innovation district.
+      We are currently running our courses out of La Commune a leading startup accelerator in downtown Montreal's Old Port.
```
Comment (ziad-saab) [inline src/about/index.html]: La Commune is a co-working space rather than a startup accelerator. Can you please reflect that in the description?

**Comment by karlsc** [comment]
This should do the trick!

**Comment by nyanofthemoon** [comment]
Looking good! If this was an official pull request, I would merge it. :+1:

**Pull request closed without merging**